src/main/lrsql/init/clamav.clj

```diff
@@ -0,0 +1,32 @@
+(ns lrsql.init.clamav
+  "ClamAV virus scanning"
+  (:require [clojure.string :as cs]
+            [clojure.java.io :as io])
+  (:import [xyz.capybara.clamav ClamavClient]
+           [xyz.capybara.clamav.commands.scan.result ScanResult ScanResult$OK]))
+
+;; questions
+;;
+;; * What do we want to know about found viruses? Anything?
+;; *
```

Comment:
This comment should be changed to a list of what is actually revealed about found viruses.

Reply:
oh sorry, this can be removed entirely.

Reply:
e8ff39808578a952c02bcf7cb4ef93ced1b1aa06